src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -752,7 +754,8 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
 
           if (itinerary.connection.egress != null) {
             val egress = option.egress.get(itinerary.connection.egress)
-            val vehicle = egressVehicles.find(v => v.mode.r5Mode.get.left.get == egress.mode).get
+            val vehicle =
```

Comment:
Same code suggestion here - also this ends in a `get` - does that need handled?

Reply:
last get is coming from Option.get but in this PR we are handing EItherGet and TryGet. will take a look how we can handle in next PR based on Option.get